GirafRest.Test/Data/SampleDataHandlerTest.cs

```diff
@@ -0,0 +1,90 @@
+﻿using GirafRest.Models;
+using GirafRest.Setup;
+using GirafRest.Test.Mocks;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+using static GirafRest.Test.UnitTestExtensions;
+
+namespace GirafRest.Test
+{
+    public class SampleDataHandlerTest
+    {
+        private readonly ITestOutputHelper _outputHelper;
+        private TestContext _testContext;
+        private const string testJson = "..\\netcoreapp2.2\\Data\\samplesTest.json";

```

Comment:
Skal denne bruge `Path.DirectorySeparatorChar` i stedet for hardkodet til `\\` som mappe-seperator?

Reply:
Og vil dette ikke fejle med reference til `netcoreapp2.2`? Når vi fx opgraderer til .netcore 3.1; Denne sti afhænger jo af compilation output path

Reply:
> 
> 
> Og vil dette ikke fejle med reference til `netcoreapp2.2`? Når vi fx opgraderer til .netcore 3.1; Denne sti afhænger jo af compilation output path

Jo, vi kendte ikke Path.DirectorySeparatorChar, så jeg ser lige på at få den ændret så den er relativ